apps/gp2-server/test/handlers/event/update-events-meeting-link-handler.test.ts

```diff
@@ -0,0 +1,200 @@
+import { when } from 'jest-when';
+
+import {
+  getCommonGoogleId,
+  meetingLinkUpdateHandler,
+} from '../../../src/handlers/event/update-events-meeting-link-handler';
+import { getEventEvent, getEventResponse } from '../../fixtures/event.fixtures';
+import { eventControllerMock } from '../../mocks/event.controller.mock';
+import { loggerMock } from '../../mocks/logger.mock';
+
+describe('getCommonGoogleId', () => {
+  test('works when it receives a series googleId event', () => {
```

Comment:
im not a big fan of the the use of words like `works` in tests - it should say what the function does, ie `should extract commong google ID from the event ID` or something like that